src/server.py

```diff
@@ -2,30 +2,71 @@
 """Server module."""
 from __future__ import unicode_literals
 import socket
+import email.utils
+
+
+def response_ok():
+    """Return 200 ok."""
+    first_line = u'HTTP/1.1 200 OK'
+    second_line = u'Content-Type: text/plain; charset=utf-8'
+    date_line = u'Date: ' + email.utils.formatdate(usegmt=True)
+    header_break = u''
+    body = u'It is all good'
+    bytes = body.encode('utf-8')
```

Comment:
`bytes` is a Python built-in function - you should probably consider using a different symbol name.